src/nn/graph_builder.ts

```diff
@@ -75,13 +76,37 @@ export enum MLAutoPad {
  * [spec](https://webmachinelearning.github.io/webnn/#dictdef-mlconv2doptions)
  */
 export interface MLConv2dOptions {
+  padding?: [number, number, number, number];
+  strides?: [number, number];
+  dilations?: [number, number];
+  outputPadding?: [number, number];
+  autoPad?: MLAutoPad;
+  groups?: number;
+  inputLayout?: MLInputOperandLayout;
+  filterLayout?: MLFilterOperandLayout;
```

Comment:
`MLFilterOperandLayout` is outdated, please change `MLFilterOperandLayout` to `MLConv2dFilterOperandLayout`, and also rename above enum `MLFilterOperandLayout` to `MLConv2dFilterOperandLayout`.